src/vcpkg/install.cpp

```diff
@@ -711,7 +711,7 @@ namespace vcpkg::Install
 
                     const auto name = cmakeify(bpgh.spec.name());
                     auto msg = Strings::concat(
-                        "The package ", bpgh.spec, " is header only and can be used from CMake via:\n\n");
+                        "The package ", bpgh.spec.name(), " is header only and can be used from CMake via:\n\n");
```

Comment:
I can see an argument that libraries may be header only only for some triplets which would argue for keeping the whole spec here, but I think that's unusual enough that for heuristics we should strip that.